providers/github/resources/github_repo.go

```diff
@@ -155,7 +155,6 @@ func initGithubRepository(runtime *plugin.Runtime, args map[string]*llx.RawData)
 				}
 				user = obj.(*mqlGithubUser)
 			}
-			return nil, nil, err
```

Comment:
was this not intended to return the err if the err is not a 404?

Reply:
Yes, but API thows 404 in the case repo owner is a user, hence the handling before this return
There is exactly same return above for the case user isn't found as well

So, if org not found -> attempt to find user -> if user not found -> return error as is

Example for non-existent repo
<img width="1037" alt="Screenshot 2024-07-02 at 18 56 52" src="https://github.com/mondoohq/cnquery/assets/32666515/61559c24-96e3-4650-a248-bc04bd113039">

Reply:
thanks for the explanation!